src/commands/networks/removeNetwork.ts

```diff
@@ -16,6 +16,11 @@ export async function removeNetwork(context: IActionContext, node: NetworkTreeIt
         nodes = await ext.networksTree.showTreeItemPicker(NetworkTreeItem.contextValue, { ...context, canPickMany: true, suppressCreatePick: true });
     }
 
+    if (nodes.some(node => ['bridge', 'host', 'none'].includes(node.networkName))) {
+        ext.ui.showWarningMessage("It's not possible to remove the built-in networks 'bridge', 'host', or 'none'");
```

Comment:
Nit: avoid contractions in messages shown to users (`'It is not possible...'`).  Alternatively, `"The built-in networks 'bridge', 'host', and 'none' cannot be removed."`